Engine/csound_orc_compile.c

```diff
@@ -1995,7 +1995,9 @@ if (engineState != &csound->engineState) {
 }
 
 #ifdef EMSCRIPTEN
-void sanitize(CSOUND *csound) {}
+//TODO: how to handle both emscripten and wasm-sdk builds?
```

Comment:
Can you explain the problem?

Reply:
When I initially created the pr the existing wasm github runner was disabled.  It shouldn't be an issue since it's enabled now.